spyder/widgets/variableexplorer/tests/test_dataframeeditor.py

```diff
@@ -174,6 +179,36 @@ def test_change_format_with_format_not_starting_with_percent(qtbot, monkeypatch)
     with qtbot.assertNotEmitted(editor.sig_option_changed):
         editor.change_format()
 
+def test_header_encoding():
+    data =  u"""a,Unieke_Idcode,ü,b,1
+            0,flooooo,0.55,10100103,31/12/2016
+            """
+    test_data = StringIO(data)
+    df = read_csv(test_data, encoding='utf-8')
+    editor = DataFrameEditor(None)
+    editor.setup_and_check(df)
+    model = editor.dataModel
+    assert model.headerData(0, orientation=Qt.Horizontal) == u"Index"
+    assert model.headerData(1, orientation=Qt.Horizontal) == u"a"
+    assert model.headerData(2, orientation=Qt.Horizontal) == u"Unieke_Idcode"
+    assert model.headerData(3, orientation=Qt.Horizontal) == u"ü"
+    assert model.headerData(4, orientation=Qt.Horizontal) == u"b"
+    assert model.headerData(5, orientation=Qt.Horizontal) == u"1"
+
+def test_header_bom():
+    df = read_csv('spyder/widgets/variableexplorer/tests/test.csv',
+                  encoding='utf-8-sig')
```

Comment:
Ok, I see. Then you can get the cwd of a file being run in Python with

    os.path.dirname(os.path.realpath(__file__))

so you can add that as a constant at the top of this file.